### PR TITLE
feat: add beehiiv posts

### LINKED ISSUE #8423: Footer replacement with beehiiv posts
Substack publications are on hold, and we have this more actual 
https://kodadotweeklyroundup.beehiiv.com/

showing last three posts from our publications is fair

https://developers.beehiiv.com/docs/v2/ab0d2f8ee91c2-show

API stuff should be somewhere around 

image

replace with this

image

### PR DESCRIPTION
**Pull request by preschian**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

## Needs Design check

- @exezbcz please review

## Context

- [x] Closes #8423

#### Did your issue had any of the "$" label on it?

- [x] My DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer/?target=1xjvRADwdJcnmUCLWerEHR4iGCT5EBTm4D4fzLLg4LcAC9p)

## Screenshot 📸

- [x] My fix has changed UI

image

### CODE REVIEW

File: script/posts.mjs [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import fetch from 'node-fetch'
+import fs from 'fs/promises'
+import consola from 'consola'
+
+async function init() {
+  try {
+    const response = await fetch('https://newsletter.kodadot.workers.dev/index')
```
Comment (roiLeo) [inline script/posts.mjs:7]: https://github.com/kodadot/workers/pull/233/files/e7d9383ec30b25c9d013cfc184da13b32b2c6793#r1433670807

**Review by roiLeo — APPROVED** [review]

**Comment by yangwao** [comment]
merged - https://github.com/kodadot/workers/pull/233/

**Comment by yangwao** [comment]
ah it's fetches from posts.json oka, so can merge?

**Comment by yangwao** [comment]
Thanks!
pay 50 usd

**Pull request merged**